assets/js/rootLantiq.js

```diff
@@ -131,3 +131,20 @@ async function unlockHuaweiShell(port, outputMsgCallback, outputErrorCallback, b
         return false;
     }
 }
+
+async function startYmodemLoad(port, outputMsgCallback, outputErrorCallback, baudRate = 115200, address = "0x80800000") {
```

Comment:
callbacks should be put at the end not in the middle

Reply:
We can't use the default parameters by moving the callbacks to the end